Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM php:7.3
+
+COPY composer.json /app/
+COPY Classes /app/Classes
+
+RUN useradd -ms /bin/bash application

```

Comment:
this container could also run under root since:
- its local
- its just a php-server and does not need special permission treatment